upgrade-tidb-using-tiup.md

```diff
@@ -45,7 +45,7 @@ This section introduces the preparation works needed before upgrading your TiDB
 
 ### Step 1: Review compatibility changes
 
-Review [the compatibility changes](/releases/release-7.1.0.md#compatibility-changes) in TiDB v7.1.0 release notes. If any changes affect your upgrade, take actions accordingly.
+Review [the compatibility changes](/releases/release-7.2.0.md#compatibility-changes) in TiDB v7.2.0 release notes. If any changes affect your upgrade, take actions accordingly.
```

Comment:
```suggestion
Review [the compatibility changes](/releases/release-7.1.0.md#compatibility-changes) in TiDB v7.1.0 release notes. If any changes affect your upgrade, take actions accordingly.
```

Reply:
will be updated in https://github.com/pingcap/docs/pull/13910/commits/c6147e31329981aa6fd01043361b8c71fab4370f instead